src/redux/auth/reducer.js

```diff
@@ -0,0 +1,23 @@
+import * as actionTypes from "./actionTypes";
+
+const authStoreInitialState = {
+  isSignIn: false
+};
+
+const authReducer = (state = authStoreInitialState, action) => {
+  switch (action.type) {
+    case actionTypes.SIGN_IN_AND_SET_JWT_SUCCESS: {
+      return { isSignIn: true };
+    }
+    case actionTypes.SIGN_OUT_AND_REMOVE_JWT_SUCCESS: {
+      return { isSignIn: false };
+    }
+    case actionTypes.JWT_VALIDATE_TURE: {
+      return { isSignIn: false };
```

Comment:
나중에 IntitailState가 늘어날 수도 있지 않을까요? 🤔

```suggestion
      return {
      ...state,
       isSignIn: false
        };
```

Reply:
InitialState가 복잡해지는 경우 ...state로 퉁치게 되면 변해야할 상태가 변하지 않거나 다른 컴포넌트에 영향을 주거나 하는 경우가 종종 있더라고요. 이런 경우 때문에 일부러 ...state를 빼봤는데 역시 넣는게 좋을까요?

Reply:
> 변해야할 상태가 변하지 않거나 다른 컴포넌트에 영향을 주거나
잘 작성하면 잘 변하지 않을까요?! ㅎㅎ 😚

지금은 이대로 충분하고 다음에 늘어날 때 넣으면 좋을거같아요 ㅎㅎ